media/css/newsletter/newsletter-opt-out-confirmation.scss

```diff
@@ -33,9 +33,15 @@ header {
     }
 
     .tagline {
-        @include font-size-level4;
+        @include font-size(18px);
```

Comment:
If we need this to match `<body>` copy size, could we just drop these size declarations altogether? Wouldn't that fall back to match the rest of the copy size?

Reply:
Yup will remove 👍